### PR TITLE
Rectified Inconsistent capitalization with z/OS console command

### PR DESCRIPTION
**Pull request by chinmaym07**
Fix: #961 
Signed-off-by: chinmaym07 <b418020@iiit-bh.ac.in>

### CODE REVIEW

**Comment by MikeBauerCA** [comment]
@t1m0thyj do you want to approve and run the workflow? I'm thinking snapshots will need to be updated for this change - could you advise?

**Review by t1m0thyj — COMMENTED** [review]
@chinmaym07 Thanks for your contribution! I've approved the GitHub workflows to run, could you please do the following to make them happy? 🙂 
* Update test snapshots by running Jest with `-u` flag
  * For unit tests: Run `npm run test:unit -- -u`
  * For integration tests it takes a long time to run all of them, so it may be easier to only update snapshots where needed. You can see the tests that had failing snapshots in the build output [here](https://github.com/zowe/zowe-cli/runs/4739281944?check_suite_focus=true). In this case, you could run `npx jest cli.zos-console.integration.test.ts -u`
* Update the packages/cli/CHANGELOG.md file following [these guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md#changelog-update-guidelines)

**Comment by chinmaym07** [comment]
Hey @t1m0thyj .. sorry for the delay here in making the workflow happy.. Hope this works now.

File: packages/cli/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+
+## `6.37.8`
```
Comment (t1m0thyj) [inline packages/cli/CHANGELOG.md:6]: Please change the header to `## Recent Changes`. Our deployment automation will update this to a version number later.

File: packages/cli/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zowe/cli",
-  "version": "6.37.7",
+  "version": "6.37.8",
```
Comment (t1m0thyj) [inline packages/cli/package.json:3]: Please revert the manual change to version number in package.json file, our automation will handle bumping it as needed.

**Comment by chinmaym07** [comment]
Hey @t1m0thyj . Can you please re-review this PR ?

**Review by t1m0thyj — APPROVED** [review]

File: packages/cli/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+
+## Recent Changes
+
+- Enhancement: Inconsistent capitalization with z/OS console command. [Issue 961](https://github.com/zowe/zowe-cli/issues/961)
```
Comment (zFernand0) [inline packages/cli/CHANGELOG.md:8]: Should wthis be a BugFix instead of an Enhancement?

**Review by zFernand0 — APPROVED** [review]

File: packages/cli/CHANGELOG.md (hunk 3 above)
Comment (t1m0thyj) [reply packages/cli/CHANGELOG.md:8]: Probably, I believe that is the category we've used previously for typo-level changes.

Comment (chinmaym07) [reply packages/cli/CHANGELOG.md:8]: Shall I update it to BugFix ?

Comment (t1m0thyj) [reply packages/cli/CHANGELOG.md:8]: Yes please 🙂

Comment (zFernand0) [reply packages/cli/CHANGELOG.md]: Thank you!  😋

**Comment by t1m0thyj** [comment]
Merging, will fix the failing audit check in #1347

**Pull request merged**